scalableminds/webknossos
# Transfer volume data zip as whole file rather than chunked

Since this wasn’t fully streamed before, I feel but little regret in making this step blocking too. This way, the requests fetching volume data from the tracingstore are sent strictly sequentially (will hopefully lessen the load on the servers), and if they fail, proper errors are thrown. Downside being that the time until first response to the user from wk is still very large for large projects (possibly even a bit longer now due to less parallelization)

### URL of deployed dev instance (used for testing):
- https://volumedataasfile.webknossos.xyz

### Steps to test:
- volume tracing download should still work for single tracing
- create a volume task type, matching project and two or more tasks
- trace some in these tasks
- in admin→projects, click download on the volume project
- should get a zip containing individual zips (which should all be valid zips, in turn containing header.wkw and buckets)
- the real bugfix can only be tested in production.

### Issues:
- an attempt at #3874 

------
